pages/test.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div>
+        <div>
+            <router-link to="/">Go Home</router-link>
+        </div>
+        <div v-if="error">
+            <p>something went wrong</p>
+            <p>{{ error }}</p>
+        </div>
+        <div v-else>
+            <br/>
+            <p>You are logged in as {{ data?.display_name }}</p>
+            <br/>
+            <p>{{ JSON.stringify(data) }}</p>
+        </div>
+    </div>
+</template>
+
+<script setup>
+    const auth_token = localStorage.getItem('auth-token');
+
+    var data;
```

Comment:
in JS bevorzugt man normalerweise "let" zu "var". var ist praktisch depricated.

Reply:
var, let und const haben unterschiedliche scopes. let würde unten (error = ...) nicht funktionieren. oben könnte man noch machen.